src/RegionSelectAndTransformBoxes/index.js

```diff
@@ -172,29 +173,31 @@ export const RegionSelectAndTransformBox = memo(
                   .inverse()
                   .applyToPoint(px * iw, py * ih)
                 return (
-                  <TransformGrabber
-                    key={i}
-                    {...mouseEvents}
-                    onMouseDown={(e) => {
-                      if (e.button === 0 && (!r.open || i === 0))
-                        return onBeginMoveKeypoint(r, keypointId)
-                      mouseEvents.onMouseDown(e)
-                    }}
-                    style={{
-                      cursor: !r.open
-                        ? "move"
-                        : i === 0
-                        ? "pointer"
-                        : undefined,
-                      zIndex: 10,
-                      pointerEvents:
-                        r.open && i === r.points.length - 1
-                          ? "none"
+                  <Tooltip title={keypointId}>
```

Comment:
this component needs a `key={i}`

Reply:
it'll throw an error in the console otherwise and possibly cause rendering issues